py_ecc/fields/field_elements.py

```diff
@@ -173,6 +178,17 @@ def __repr__(self: T_FQ) -> str:
     def __int__(self: T_FQ) -> int:
         return self.n
 
+    def __lt__(self: T_FQ, other: IntOrFQ) -> bool:
```

Comment:
Should `other` be `Any` here to accommodate the `else`?
```suggestion
    def __lt__(self: T_FQ, other: Any) -> bool:
```

Reply:
The only dunder methods that complained about not being `Any` are `__eq__` and `__ne__`. All the typed examples I can find of comparison dunders have `other` being of the same type as `self`. This seems to be a special case where both `FQ` and `int` are kind of considered the same.

Here's a similar example from `pycryptodome` - https://github.com/Legrandin/pycryptodome/blob/13a8461e068ebca99d21e0f2f71830504b25ac7e/lib/Crypto/Math/_IntegerBase.pyi#L17